<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Watcher</class>
 <widget class="QWidget" name="Watcher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1068</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Watcher</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,10,0,1" columnstretch="1,4">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="SyncCheckBox">
         <property name="text">
          <string>Sync</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="watcherSelect">
         <item>
          <property name="text">
           <string>AD</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="3">
        <widget class="QStackedWidget" name="stackedWidget_2">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout" stretch="10,0">
             <item>
              <widget class="QWidget" name="subplt2" native="true"/>
             </item>
             <item>
              <widget class="QSlider" name="subSlider2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="subtitle2">
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QTextEdit" name="text2"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="plt_page">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2" stretch="10,0">
             <item>
              <widget class="QWidget" name="subplt1" native="true"/>
             </item>
             <item>
              <widget class="QSlider" name="subSlider1">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="subtitle1">
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="text_page">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QTextEdit" name="text1"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="CloseButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>CloseButton</sender>
   <signal>pressed()</signal>
   <receiver>Watcher</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>451</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
